apps/common/lib/lexical/ast/analysis.ex

```diff
@@ -116,6 +116,20 @@ defmodule Lexical.Ast.Analysis do
     end)
   end
 
+  def module_scope(%__MODULE__{} = analysis, %Position{} = pos) do
+    enclosing_scopes = scopes_at(analysis, pos)
+
+    first_scope = List.first(enclosing_scopes)
+
+    Enum.reduce_while(enclosing_scopes, first_scope, fn
+      %Scope{module: same} = current, %Scope{module: same} ->
+        {:cont, current}
+
+      _, current ->
+        {:halt, current}
+    end)
+  end
+
```

Comment:
Unless I'm misunderstanding this function, I don't think it's needed. Look for `Analyzer.current_module(analysis, position)` as a replacement.

Reply:
I think this one still needs to be removed!